components/has/kustomization.yaml

```diff
@@ -1,10 +1,15 @@
 resources:
 - allow-argocd-to-manage.yaml
-- catalogsource.yaml
-- operatorgroup.yaml
-- subscription.yaml
+- https://github.com/redhat-appstudio/application-service/config/default
```

Comment:
Can (or is it possible to?) we point to a specific ref in https://github.com/redhat-appstudio/application-service too?

Reply:
Yes.
Like this
github.com/Liujingfang1/kustomize/examples/helloWorld?ref=repoUrl2

Reply:
- https://github.com/redhat-appstudio/application-service/config/default?ref=e97f8514f0552df4bb92d4bc7133e6991f77af97